src/BrandedNavBar/NavBar.tsx

```diff
@@ -23,35 +23,22 @@ const themeColorObject = {
   logoColor: "blue",
 };
 
-type MediumNavBarProps = BrandLogoContainerProps & {
-  subtext?: string;
+type MediumNavBarProps = {
   menuData?: any;
   environment?: "development" | "training";
+  logo: React.ReactElement;
```

Comment:
I think we want to use `React.ReactNode`

Reply:
@haideralsh is that because `React.ReactNode` allows a list of `React.ReactElement`s?